masterlist.yaml

```
@@ -4205,8 +4205,9 @@ plugins:
       - crc: 0xD0F78346
         util: 'SSEEdit v4.0.2f'
 
-  - name: 'KaliliesNPCs.esp'
+  - name: 'Kalilies( )?NPCs( \+ AI Overhaul)?( - CRF| \+ Cutting Room Floor)?( Patch)?.esp'
```

Maybe this instead, a bit easier to look at.
```suggestion
  - name: '(KaliliesNPCs|Kalilies NPCs).*\.esp'
```

---

I avoided that since it would include the plugin on [this page](https://www.nexusmods.com/skyrimspecialedition/mods/33942/), `KaliliesNPCs WARP PATCH.esp`. I just figured out how to exclude phrases from a regex, so I'll commit an adjustment for you to take a look at.

---

Makes sense, but the same thing could happen for any patch hosted on a different page.
Worst case is it's going to have the original mod and patch listed in locations.

---

Yeah, that's just the only one I'm aware of and that's linked to on Kalilies' page, so I wanted to take it into consideration. Thankfully, I found a much more elegant solution than my previous one.

---

It works and isn't hard to read, I'm happy enough with that.